src/app/workshop-details/workshop-details.component.html

```diff
@@ -65,7 +65,7 @@ <h2 class="highlight-headings">Price</h2>
                 <h2 class="highlight-headings">Workshop Link(s)</h2>
                 <div class="dates-wrapper">
                     <div class="dates-information" *ngFor="let detail of workshopDetails.multiWorkshopDetails;">
-                        <a href="{{detail.link}}" target="_blank">
```

Comment:
did we add for photographer website link?

Reply:
will do